guideml.c

```diff
@@ -908,127 +922,164 @@ if((param.smartwrap) && (strlen(buf)<2))
           link++;
           while(*link && (*link==' ' || *link==',' || *link=='\t')) link++;
           if(!*link) return(0);
-          if(Strnicmp(link,"link",4))
```

Comment:
Functionnal change: I have added some handling of the "system" command when it is used with the "openurl". This is used in the ace.cpcscene.net userguide for links to external websites. This code converts them to html links to the corresponding URLs.

The code is a bit incomplete, in particular it does not check if "system" is used with something else than the "openurl" command and will probably generate invalid links in that case.

Reply:
Some checks to see if it is using openurl would be useful here, although I'm not sure the system command sees much use.

Reply:
I added a check for openurl, in other cases the command is ignored with a warning message and no link is generated.